cluster/environments/azure-multiple-clusters-waf-tm-apimgmt/waf-westus.tf

```diff
@@ -1,17 +1,17 @@
 module "west_waf_subnet" {
-  source = "github.com/microsoft/bedrock?ref=master//cluster/azure/subnet"
```

Comment:
Does this template test run with relative paths? I am thinking the provider used when it was created may have a conflict with the current azurerm provider.